demo-app/src/app/validation/validation-binding/validation-binding.component.html

```diff
@@ -37,9 +37,9 @@
               [formControlDisabled]="form.controls['checkbox'].value"
               [label]="'Select'"
               [placeholder]="'Choose an option'">
-              <option>Option 1</option>
-              <option>Option 2</option>
-              <option>Option 3</option>
+              <option [value]="option1">Option 1</option>
+              <option [value]="option2">Option 2</option>
+              <option [value]="option3">Option 3</option>
```

Comment:
Is it really necessary to specify `[value]="..."`knowing when there is no value it is supposed to take the text as the value and in this case both are the same?